grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -67,6 +68,20 @@
     private static final Set<SerializationFormat> DEFAULT_SUPPORTED_SERIALIZATION_FORMATS =
             GrpcSerializationFormats.values();
 
+    private static final boolean USE_COROUTINE_CONTEXT_INTERCEPTOR;
+
+    static {
+        boolean useCoroutineContextInterceptor;
+        try {
+            Class.forName("io.grpc.kotlin.CoroutineContextServerInterceptor", false,
+                          GrpcServiceBuilder.class.getClassLoader());
+            useCoroutineContextInterceptor = true;
+        } catch (Throwable ignored) {
+            useCoroutineContextInterceptor = false;
+        }
+        USE_COROUTINE_CONTEXT_INTERCEPTOR = useCoroutineContextInterceptor;
```

Comment:
How about logging whether coroutine context interceptor is available? e.g.

```java
logger.debug("{}: {}", className, useCoroutineContextInterceptor ? "available" : "unavailable");
```

Reply:
Yeah. That's a good idea!